docker/Dockerfile.firedrake

```diff
@@ -1,29 +1,30 @@
-FROM firedrakeproject/firedrake-vanilla
+FROM ghcr.io/mesh-adaptation/petsc-parmmg
 
 USER firedrake
 WORKDIR /home/firedrake
 
-# Reconfigure PETSc with packages required for mesh adaptation and rebuild
-RUN bash -c "cd petsc && \
-	  git fetch && \
-	  cp default/lib/petsc/conf/reconfigure-default.py . && \
-	  sed -i 's/]/]\n  configure_options.extend(sys.argv[1:])/g' reconfigure-default.py && \
-	  ./reconfigure-default.py $(echo -n $(curl https://raw.githubusercontent.com/mesh-adaptation/mesh-adaptation-docs/main/install/petsc_reconfigure_options.txt)) && \
-    make all"
+ENV PETSC_ARCH default
```

Comment:
This and other `ENV` lines raise a warning now: `LegacyKeyValueFormat: "ENV key=value" should be used instead of legacy "ENV key value"`

Reply:
Hm this must be only for recent Docker installations because the version on my machine doesn't support it. Let's leave it for now.